Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ocaml/opam2:4.07
+
+# Set home directory
+ARG HOME=/home/opam
+
+# Create working directory
+WORKDIR $HOME/lama
+
+# Copy files
+COPY --chown=opam:opam . .
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Configure and build project
+RUN    sudo apt-get update \
```

Comment:
I'd recommend you te separate dependencies & build phase, it allows docker to cache dependencies (which I assume change much less frequently than the source code of the project itself)